src/LinuxTracing/TracerImpl.cpp

```diff
@@ -613,7 +654,8 @@ bool TracerImpl::OpenGpuTracepoints(const std::vector<int32_t>& cpus) {
        {"amdgpu", "amdgpu_sched_run_job", &amdgpu_sched_run_job_ids_},
        {"dma_fence", "dma_fence_signaled", &dma_fence_signaled_ids_}},
       cpus, &tracing_fds_, GPU_TRACING_RING_BUFFER_SIZE_KB, &gpu_tracepoint_ring_buffer_fds_per_cpu,
-      &ring_buffers_);
+      &ring_buffers_, CaptureOptions::kNoThreadStateChangeCallStackCollection,
```

Comment:
Consider giving those parameters a default value, such that you don't need to pass them if not interested.